### ISSUE TITLE
MultitaskClassifier with ConvMolFeaturizer

### ISSUE THREAD

**Issue by albertma-evotec** [deepchem/deepchem#2285]
Does the MultitaskClassifier work with ConvMolFeaturizer?

I see that MultitaskClassifier object requires the n_features parameter but the ConvMolFeaturizer does not seem to create n-dimensional features.

Am I missing something here or these two thing are not compatible to each other? Is there a way to extract any numerical features after featurizing my molecules using the ConvMolFeaturizer?

Thanks

**Comment by peastman** [comment]
No, they are not compatible.  ConvMolFeaturizer is a specialized featurizer for use with graph convolutions.  For a model that takes a vector of numbers, use a CircularFingerprint as the featurizer.

**Comment by albertma-evotec** [comment]
> No, they are not compatible. ConvMolFeaturizer is a specialized featurizer for use with graph convolutions. For a model that takes a vector of numbers, use a CircularFingerprint as the featurizer.

Thanks for prompt reply. Is this CircularFingerprint same as MorganFingerprint in rdkit? 
I was trying the dc.feat.RDKitDescriptors() as the featurizer which gave me X of 200-dimensional feature. Is there a convenience way I can use both the RDKitDescriptors and the CircularFingerprint as featurizers to give me a (2048+200)-dimensional X?
